src/chunkserver/datastore/define.h

```diff
@@ -34,10 +34,14 @@ namespace chunkserver {
 
 using curve::common::Bitmap;
 
+// In zeroed chunk file, taking endianness into consideration, the version is 8,
+// Taking endianness into consideration, the version is 4,
 // In zeroed chunk file, the version is 2,
 // otherwise, the version is 1
 const uint8_t FORMAT_VERSION = 1;
 const uint8_t FORMAT_VERSION_V2 = 2;
+const uint8_t FORMAT_VERSION_V3 = 4;
+const uint8_t FORMAT_VERSION_V4 = 8;
```

Comment:
why not  is 3  and 4 ?

Reply:
It's all right for 3 and 4, maybe I just set it as 2^(n-1) at that time, should i fix it?